### PR TITLE
Adds Ramadan.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Ramadan changes every year. You can't just make it apply for one year.
Speaking of which, it lasts until 24th of June

**Comment by ghost** [comment]
Yes, that's why I titled it for the beginning day of the holiday. I'm not sure how to alter the holiday code so that it varies year to year, whoever originally made it left a note in the comments saying as much. I figure this PR gives me about a year to figure out how to optimize our new Ramadan feature.

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -179,7 +179,20 @@
 /datum/holiday/firefighter/getStationPrefix()
 	return pick("Burning","Blazing","Plasma","Fire")
 
-/datum/holiday/summersolstice
+/datum/holiday/ramadanBegins
```
Comment (GuyonBroadway) [inline code/modules/holiday/holidays.dm:182]: this needs to go in our hippiestation module

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

**Comment by tortellinitony** [comment]
This won't work.

**Comment by tortellinitony** [comment]
Yeah, unless you can actually figure out an accurate way of mapping when Ramadan starts based on the lunar calendar, there is no reason to implement this.

**Comment by ghost** [comment]
That's not true, Tony...!

**Pull request closed without merging**